docs/src/library/bethe-free-energy.md

```diff
@@ -1,5 +1,71 @@
 # [Bethe Free Energy implementation in RxInfer](@id lib-bethe-free-energy)
 
+The following text aims to introduce the Bethe Free Energy with as little mathematics as needed (but no less). We start be defining a factorized model and move from the Variational Free Energy to a definition for the Bethe Free Energy.
+
+## [Factorized model](@id lib-bethe-factorized-model)
+
+Before we can define a model, we must identify all variables that are relevant to the problem at hand. We distinguish between variables that can be directly observed,
+
+$$y = (y_1, \dots, y_j, \dots, y_m)$$
+
+and variables that can not be observed, which are also named latent variables.
+
+$$x = (x_1, \dots, x_i, \dots, x_n)$$
+
+We then define a model that factorizes over consituent smaller factors (functions).
+
+$$f(y,x) = \prod_a f_a(y_a,x_a)$$
+
+Individual factors may represent stochastic functions, such as conditional or prior distributions, but also potential functions or deterministic relationships. A factor may depend on multiple observed and/or latent variables (or none).
+
+
+## [Variational Free Energy](@id lib-bethe-vfe)
+
+The Variational Free Energy (VFE) defines a functional objective that includes the model and a variational distribution over the latent variables.
+
+$$F[q](\hat{y}) = \mathbb{E}_{q(x)}\left[\log \frac{q(x)}{f(y=\hat{y}, x)} \right]$$
+
+A functional is the mathematical term for a function of a function that returns a scalar.
```

Comment:
Are these spaces between the sentences intentional? Markdown starts a new paragraph if there is a blank line (as Latex)

Reply:
![Screenshot 2023-10-30 at 10 50 12](https://github.com/biaslab/RxInfer.jl/assets/6557701/eb6fb308-bc43-4386-ac3a-eb0f5eebe685)